xtask/src/main.rs

```diff
@@ -89,8 +89,8 @@ fn run_action(workspace: Workspace, target: &str, extra_args: &[String]) {
                 let tracing_kind = find_tracing_kind(&rust_flags);
                 rust_flags = make_internal_rustflags(&rust_flags);
 
-                // Optimise the internal workspace. Don't pass `--release` for `cargo bench` however as it is
-                // already implied (and explicitly passing it is illegal).
+                // Optimise the internal workspace. Don't pass `--release` for `cargo bench`
```

Comment:
On reflection, I think it's fairly obvious that `--release` implies optimisation, so maybe we can simplify this to something like "`cargo bench` errors if you pass `--release`"?

Reply:
> On reflection, I think it's fairly obvious that --release implies optimisation

Your proposed text is fine, but not that the subtlety is that `cargo bench` implies `--release`, not that `--release` implies optimisation.

[Maybe you meant that?]

Reply:
"cargo bench implies --release" yes "not that --release implies optimisation" um no?

Reply:
I think you misunderstood.

The fact that --release implies optimisation, whilst true, is irrelevant.

What matters here is that `cargo bench` implies optimisation, and this is the root cause of why `--release` is invalid in that context.

Reply:
We can be more concise with:
```
// Optimise the internal workspace. `--release` is not valid for `cargo bench`.
```

Reply:
I think you are misunderstanding me :) But since I'm happy with your proposed new text (it's entirely compatible with my understanding), I don't think we should worry about it. Let's use your newest suggestion.

Reply:
OK, so e9556f2 fixes that.